TeamForward/client/src/components/ProfilePage/Profile.js

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useState } from "react";
+import { userState } from "../../GlobalState";
+import { Link } from "react-router-dom";
+import ConnectButton from "../Button/ConnectButton";
+
+const Profile = ({ profileData, setProfileData }) => {
+  const user = userState();
+  // const user = useReactiveVar(userState);
```

Comment:
use useReactiveVar instead of userState();
otherwise this component won't update when userState() updates - per James